docs/UserGuide.md

```diff
@@ -93,20 +99,54 @@ Figure 3: Help Summary Window
 
 
 
-#### 3.1.2 Adding an item: `add-i` or `add-d`
+#### 3.1.2 Adding an item or delivery: `add-i` or `add-d`
 
 Adds inventory item or pending delivery to OneShelf.
 
-Format: `add-i n/NAME q/QUANTITY [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]...​`
+##### 3.1.2.a `add-i`
+
+Format: `add-i n/NAME q/QUANTITY [s/SUPPLIER] [max/MAX_QUANTITY] [metric/METRIC] [t/TAG]...​`
 * Adds `QUANTITY` from the current quantity of an inventory item if the item already exist.
-An item is uniquely identified by its NAME and SUPPLIER.
+An item is uniquely identified by its `NAME` and `SUPPLIER`.
+* Note 1: `MAX_QUANTITY` denotes the ideal stock level of that particular item.
+* Note 2: `TAG` could be used to categorise items. EG: Duck can be tagged as meat. 
+
+Example 1: `add-i n/DUCK q/10 s/NTUC max/50 metric/KG` using [Figure 1](#bruiimagesuiwithannotationpng-br) as a base.
```

Comment:
If i were u i'll add multiple tags too for full set, then example 2 is minimum add = only name and quantity

Reply:
Got it!